clang/test/Analysis/non_loc_compound.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_analyze_cc1 %s \
```

Comment:
I think you should put this test without the RUN lines into the `clang/test/Analysis/invalidated-iterator.cpp` to have them at one place.